plugins/modules/list_clusters.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+import requests
+
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: list_clusters
+
+short_description: lists ocp clusters
+
+version_added: "1.0.0"
+
+description: Given the API token, returns list of clusters in the account
+
+options:
+    fail:
+        description: Trigger a failure
+        default: false
+        type: bool
+
+author:
+    - Name (@vjayaramrh)
+"""
+
+EXAMPLES = r"""
+# Use argument
+- name: Trigger a failure
+  list_clusters
+    fail: true
+"""
+
+RETURN = r"""
+list_clusters_msg:
+    description: A message with the status
+    type: str
+    returned: always
+    sample: "Success!"
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+BASE_URL = "https://api.openshift.com/api/assisted-install/v2/clusters"
+
+def get_token():
+    ai_api_token = os.environ.get('AI_API_TOKEN')
+    if not ai_api_token:
+        return "error"
```

Comment:
Should handle a bit better the failure. You probably want to avoid returning a string with error.
Perhaps this is not needed after all, wdyt?

Reply:
Agree that the get_token() is not needed